scripts/fetchdep.sh

```diff
@@ -17,10 +17,8 @@ clone() {
     fi
 }
 
-# Try the PR author's branch in case it exists on the deps as well.
-clone $TRAVIS_PULL_REQUEST_BRANCH
```

Comment:
It looks like `$BUILDKITE_BRANCH` is the PR author's branch, while `$BUILDKITE_PULL_REQUEST_BASE_BRANCH` is the target branch. I think we can use both to preserve current behaviour here.